mapfollower:
  init_takeoff: 0.8
  lidar_avoidance_distance: 1.0
  global_pose_x_min: -1.7
  global_pose_y_min: -1.7
  global_pose_z_min: 0.0
  global_pose_x_max: 1.7
  global_pose_y_max: 1.7
  global_pose_z_max: 2.0
